vimeo-networking/src/main/java/com/vimeo/networking/callers/GetRequestCaller.java

```diff
@@ -0,0 +1,472 @@
+                }
+            };
+
+    /**
+     * Used in association with
+     * {@link VimeoClient#getContent(String, CacheControl, VimeoCallback, String, Map, String, Caller)} or
+     * {@link VimeoClient#getContentSync(String, CacheControl, String, Map, String, Caller)}
+     * to get a {@link Void} response from an API endpoint.
+     */
+    public static final Caller<Void> VOID =
+            new Caller<Void>() {
+
+                @NotNull
+                @Override
+                public Call<Void> call(@NotNull String authHeader,
+                                       @NotNull String uri,
+                                       @NotNull Map<String, String> queryMap,
+                                       @NotNull String cacheHeader,
+                                       @NotNull VimeoService vimeoService) {
+                    return vimeoService.getVoid(authHeader, uri, queryMap, cacheHeader);
+                }
+            };
+
+    /**
+     * Used in association with
+     * {@link VimeoClient#getContent(String, CacheControl, VimeoCallback, String, Map, String, Caller)} or
+     * {@link VimeoClient#getContentSync(String, CacheControl, String, Map, String, Caller)}
+     * to get a {@link UserList} response from an API endpoint.
+     */
+    public static final Caller<UserList> USER_LIST =
+            new Caller<UserList>() {
+
+                @NotNull
+                @Override
+                public Call<UserList> call(@NotNull String authHeader,
+                                           @NotNull String uri,
+                                           @NotNull Map<String, String> queryMap,
+                                           @NotNull String cacheHeader,
+                                           @NotNull VimeoService vimeoService) {
+                    return vimeoService.getUserList(authHeader, uri, queryMap, cacheHeader);
+                }
+            };
+
+    /**
+     * Used in association with
+     * {@link VimeoClient#getContent(String, CacheControl, VimeoCallback, String, Map, String, Caller)} or
+     * {@link VimeoClient#getContentSync(String, CacheControl, String, Map, String, Caller)}
+     * to get a {@link VideoList} response from an API endpoint.
+     */
+    public static final Caller<VideoList> VIDEO_LIST =
+            new Caller<VideoList>() {
+
+                @NotNull
+                @Override
+                public Call<VideoList> call(@NotNull String authHeader,
+                                            @NotNull String uri,
+                                            @NotNull Map<String, String> queryMap,
+                                            @NotNull String cacheHeader,
+                                            @NotNull VimeoService vimeoService) {
+                    return vimeoService.getVideoList(authHeader, uri, queryMap, cacheHeader);
+                }
+            };
+
+    private GetRequestCaller() {}
```

Comment:
This class seems like it would be a great candidate for an annotation processor for generating code. @anthonycr, next weekend project :) ?????